src/applications/personalization/search-representative/config/basic-information/basicInformation.js

```diff
@@ -0,0 +1,34 @@
+export const schema = {
+  type: 'object',
+  title: 'What kind of representative are you looking for?',
+  properties: {
```

Comment:
At some point, either now or later, we'll need to structure this schema in the same way we made the [686 options selection](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/cdc43792b240194fb02b270123e845bda3e74bf1/src/schemas/686c-674/schema.js#L115-L136) to ensure we can validate as a boolean group where at least one is required.

Reply:
Yeah, I was gonna wait to see if there are any changes to the mockup before we determine the final structure. Definitely need to circle back on this later.